contact.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+    	<meta charset="utf-8">
+    	<meta name="description" content="The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.">
+
+		<title>Sign Up For Relaxr</title>
+			<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
+		<link href='http://fonts.googleapis.com/css?family=Open+Sans:300italic,300,600,800' rel='stylesheet' type='text/css'>
+
+		<!-- external CSS link -->
+		<link rel="stylesheet" href="css/reset.css">
+		<link rel="stylesheet" href="css/style.css">
+
+	</head>
+	<body>
+		<header>
+			<div class="container">
+				<h1 class="logo">Relaxr</h1>
+				<nav>
+					<ul>
+						<li><a href="index.html">Home</a></li>
+						<li><a href="blog.html">Blog</a></li>
+						<li><a href="contact.html">Sign Up</a></li>
+					</ul>
+				</nav>
+			</div>
+		</header>
+		<section id="main">
+			<div class="container formDiv">
+				<h2> Sign Up for Relaxr!</h2>
+				<form id="contactForm">
+					<input type="text" placeholder="First & Last Name" name="name" />
+					<input type="email" placeholder="Email Address" name="email" />
+					<input type="password" placeholder="Password" name="password" />
+					<input type="password" placeholder="Confirm Password" name="password" />
```

Comment:
For password inputs (or other text type inputs like `text` or `email`), we should avoid using the same value for the "name" attribute. So here, we should probably do something like:
```
<input type="password" placeholder="Password" name="password" />
<input type="password" placeholder="Confirm Password" name="confirmPassword" />
```
This gives us the ability to select them (and their user inputted values), separately:

```
var password = $('input[name="password"]').val();
var confirmPassword = $('input[name="confirmPassword"').val();

// Make sure passwords match
if (password !== confirmPassword) {
  // display error
}
```